### PR TITLE
Graphite Exporter

### PR DESCRIPTION
**Pull request by vcasadei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,102 @@
...
+
+The _OpenCensus Graphite Stats Exporter for Go_ is a package that
+exports data to [Graphite](https://graphiteapp.org/), a real-time graphing system that stores and renders graphs of received numeric time-series data on demand.
+
+## Quickstart
+
+### Import
+
+```
+import "go.opencensus.io/exporter/graphite"
```
Comment (ramonza) [inline README.md]: Fix import path

**Comment by ramonza** [comment]
Here are the remaining issues picked up by golint:
```
golint ./...
client/client.go:65:1: comment on exported method Graphite.SendMetric should be of the form "SendMetric ..."
client/client_test.go:70:9: if block ends with a return statement, so drop this else and outdent its block
client/metric.go:8:6: exported type Metric should have comment or be unexported
```

**Comment by ramonza** [comment]
@vcasadei I have created a vanity import for this exporter, would you mind changing all the import paths to: "contrib.go.opencensus.io/exporter/graphite"?

File: README.md (hunk 1 above)
Comment (vcasadei) [reply README.md]: Done

File: graphite.go [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+		for _, b := range buckets {
+			i := indicesMap[b]
+			cumCount += uint64(data.CountPerBucket[i])
+			path.Reset()
+			names := []string{sanitize(e.opts.Namespace), sanitize(vd.View.Name), "bucket"}
+			path.WriteString(buildPath(names))
+			path.WriteString(fmt.Sprintf(";le=%.2f", b))
+			path.WriteString(tagValues(row.Tags))
+			metric, _ := newConstMetric(path.String(), float64(cumCount))
+			sendRequest(e, metric)
```
[the remark this answers is not in the document]
Comment (vcasadei) [reply graphite.go]: We should not. This should be async. I fixed it.

File: graphite.go [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+type Options struct {
+	// Host contains de host address for the graphite server
+	// The default value is "127.0.0.1"
+	Host string
+
+	// Port is the port in which the carbon endpoint is available
+	// The default value is 2003
+	Port int
+
+	// Namespace is optional and will be the first element in the path
```
[the remark this answers is not in the document]
Comment (vcasadei) [reply graphite.go]: Done

File: README.md [hunk 4]
```diff
@@ -1 +1,98 @@
...
+```
+
+The API of this project is still evolving.
+The use of vendoring or a dependency management tool is recommended.
+
+### Prerequisites
+
+OpenCensus Go libraries require Go 1.8 or later.
```
[the remark this answers is not in the document]
Comment (vcasadei) [reply README.md]: Done

File: graphite.go [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+			names := []string{sanitize(e.opts.Namespace), sanitize(vd.View.Name), "bucket"}
+			path.WriteString(buildPath(names))
+			path.WriteString(fmt.Sprintf(";le=%.2f", b))
+			path.WriteString(tagValues(row.Tags))
+			metric, _ := newConstMetric(path.String(), float64(cumCount))
+			sendRequest(e, metric)
+		}
+		path.Reset()
+		names := []string{sanitize(e.opts.Namespace), sanitize(vd.View.Name), "bucket"}
+		path.WriteString(buildPath(names))
```
[the remark this answers is not in the document]
Comment (vcasadei) [reply graphite.go]: I tried to simplify in the latest commit.

**Comment by vcasadei** [comment]
@ramonza I updated the import to "contrib.go.opencensus.io/exporter/graphite" and fixed the golint issues

**Pull request merged**